extension/package.json

```diff
@@ -37,6 +37,7 @@
     "async-mutex": "^0.3.2",
     "axios": "^0.27.2",
     "bls-wallet-clients": "0.6.0",
+    "browser-passworder": "^2.0.3",
```

Comment:
This is the package metamask uses.  I can look into if there are others if we want to follow this approach

Reply:
this works